src/golang/lib/repos/operator_result.go

```diff
@@ -22,11 +22,11 @@ type operatorResultReader interface {
 	// GetBatch returns the OperatorResults with IDs.
 	GetBatch(ctx context.Context, IDs []uuid.UUID, DB database.Database) ([]models.OperatorResult, error)
 
-	// GetByDAGResultAndOperator returns the OperatorResult given the operatorID and workflowDAGResultIDs.
-	GetByDAGResultAndOperator(ctx context.Context, workflowDAGResultIDs, operatorID uuid.UUID, DB database.Database) (*models.OperatorResult, error)
+	// GetByDAGResultAndOperator returns the OperatorResult for the DAGResult and Operator specified.
+	GetByDAGResultAndOperator(ctx context.Context, dagResultID, operatorID uuid.UUID, DB database.Database) (*models.OperatorResult, error)
 
-	// GetByDAGResults returns the OperatorResult given workflowDAGResultIDs.
-	GetByDAGResults(ctx context.Context, workflowDAGResultIDs []uuid.UUID, DB database.Database) ([]models.OperatorResult, error)
+	// GetByDAGResultBatch returns all OperatorResults for the DAGResults specified.
+	GetByDAGResultBatch(ctx context.Context, dagResultIDs []uuid.UUID, DB database.Database) ([]models.OperatorResult, error)
```

Comment:
Is the naming convention to append Batch to anything that returns a list? I think GetBatch[further description] might be better.

Reply:
I'd prefer to leave it with `GetXBatch` since the others were implemented like that